schwab_api/schwab.py

```diff
@@ -479,6 +489,53 @@ def get_account_info_v2(self):
 
         return account_info
 
+    def get_lot_info_v2(self, account_id, security_id):
+        """
+        Gets info on the lots for a given position.
+
+        account_id (int) - The account where the position is.
+        security_id (int) - A reference to an aggregate position in a security
+        for an account. Obtainable from position["security_id"]. This is an
+        account-specific unique ID, not the CUSIP or ticker.
+
+        Returns None if the response code wasn't success (207 or 200), or a dict
+        with the following structure:
+        {
+            'isCostFullyKnown': bool,
+            'isGainLossFullyKnown': bool,
+            'lotErrorMessages': [{'errorCode': '39',
+                                'message': 'LATEST PRICE QUOTES USED'}],
+            'lotPrice': float,
+            'lots': [{'costBasis': float,
+                    'costPerShare': float,
+                    'gainLoss': float,
+                    'gainLossPercent': float,
+                    'isEdited': bool,
+                    'isLongTerm': bool,
+                    'lotId': string,
+                    'marketValue': float,
+                    'openDate': date,
+                    'quantity': float}, ...],
+            'lotsCount': int,
+            'totalCostBasis': float,
+            'totalGainLossDollar': float,
+            'totalGainLossPercent': float,
+            'totalLotsQuantity': float,
+            'totalMarketValue': float
+        }
+        """
+        self.update_token("api")
+        self.headers["schwab-client-ids"] = str(account_id)
+        r = requests.get(
+            f"https://ausgateway.schwab.com/api/is.Holdings/V1/Lots", params={
+                "isLong": True, "itemissueid": security_id}, headers=self.headers)
+        if r.status_code not in [200, 207]:
+            # the
+            print(
```

Comment:
Could we remove print statements from API method calls? Instead, I'd prefer to return something that indicates a failure for the caller to handle